setup.py

```diff
@@ -59,7 +59,7 @@
 			"Topic :: Utilities",
 		],
 	
-	packages = ['cinje'],
+	packages = find_packages(),
```

Comment:
`find_packages` use has been deprecated? If this corrects the file ending issue that line number mapping triggers, there may be a larger issue?

Official documentation [doesn't suggest using this function](https://docs.python.org/3.8/distutils/setupscript.html), which was an older Python 2 approach and theoretical way to "save time" back when it was most typical to have an executable `setup.py` script invoking `setuptools`. (See also: old-style namespaces which have a `__init__.py` files declaring the namespace. See [this documentation for a complete description of "native namespace packages" v. "legacy namespace packages"](https://packaging.python.org/en/latest/guides/packaging-namespace-packages/#creating-a-namespace-package). That's also now [just static project metadata](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/).)

True, though, that packages should probably be: `['cinje', 'cinje.std', 'cinje.block', 'cinje.inline']`